src/lib/api/posts/posts.query-builder.ts

```diff
@@ -0,0 +1,28 @@
+import { PostsGetManyQueryParams } from "@/lib/api/posts/posts.types";
+
+export function buildPostsQueryString(
```

Comment:
This function looks complicated, think we should add some tests for that. 😊
But I'm OK if we do it later. 😊